### PR TITLE
added new links and youtube videos

### PR DESCRIPTION
**Pull request by udhaya10**

### CODE REVIEW

File: level-1.md [hunk 1]
```diff
@@ -1,14 +1,58 @@
-### Level 1 
+## Basic Personel Finance ##
```
Comment (dingusagar) [inline level-1.md]:
```suggestion
## Basic Personal Finance ##
```

**Comment by kishaningithub** [comment]
Why not put this in README.md ?

**Comment by Madusudanan** [comment]
LGTM. I'll start putting book reviews as well.

**Review by Madusudanan — APPROVED** [review]

**Comment by Madusudanan** [comment]
> Why not put this in README.md ?

The idea is to put everything first. We can re-organize stuff later is what I feel.

**Review by vimal1083 — APPROVED** [review]

**Pull request merged**